src/libcrun/linux.c

```diff
@@ -1611,7 +1611,16 @@ libcrun_create_dev (libcrun_container_t *container, int devfd, int srcfd,
   int rootfsfd = get_private_data (container)->rootfsfd;
   const char *rootfs = get_private_data (container)->rootfs;
   size_t rootfs_len = get_private_data (container)->rootfs_len;
-  const char *rel_dev = relative_path_under_dev (device->path);
+  // Normalize path by removing trailing slash if it exists.
+  cleanup_free char *normalized_path = xstrdup (device->path);
```

Comment:
Nit. Even though it's an existing code.

This could use the `fullname` string pointer here.